contents/founders/negotiate-software-better.md

```diff
@@ -0,0 +1,91 @@
+
+# Play the sales cycle
+
+Many software contract amounts are set by a salesperson trying to hit quota. If you know the end of their quarter, you can wait until that point to get a juicy discount.
+
+There are a couple of reasons why this happens:
+
+- Companies have quarterly revenue targets set internally. Public companies have quarterly earnings reports to make too, which affect their share price. 
+- Salespeople often get paid commission quarterly or monthly - and they can get paid an ‘[accelerator](https://www.everstage.com/resources/glossary/sales-accelerator)’ for exceeding targets, so they are incentivized to cram as much as possible into a single quarter. 
+
+Salespeople aren’t totally making up the prices of course - they have to work within certain limits set by their company. But they still have a surprising amount of flexibility! 
+
+If you ask for a discount and your salesperson makes a big deal about having to get a ‘special exception made’ or ‘needing finance approval’, this is probably BS. They will probably quickly ping their manager to check it’s ok then deliberately wait a couple of days before coming back to you so it sounds like they had to fight really hard for you.
+
+Oh and that training and implementation line item that has a $10k cost which they have so kindly comped you? Yeah, that was always going to be free anyway. 
+
+> The PostHog way: transparent [pricing](/pricing) helps a _lot_ here. We are also up front about exactly [how we pay commission](/handbook/growth/sales/how-we-work#how-commission-works) - no accelerators in there, so the price you get on March 31st is exactly the same as what you get on April 1st. 
+
+# Buy in bulk to lock in a better rate
+
+Pretty much every Saas company now will give you some sort of discount for an up-front commitment of some sort - it helps their revenue retention, which is an extremely important metric for the business. 
+
+This commitment can be time-based (e.g. 1 year contract) or usage-based (e.g. X seats or Y volume). A couple of things here:
+
+- Some companies will default to a 2 or 3 year contract, so it feels like you have to negotiate to get them down to 1 year. This is bogus - assume you can get the 1 year contract at the 2 year price if that’s what they open with. 
+- _After_ you have agreed to a 1 year contract, ask if you can pay quarterly or monthly instead. If you decide part way through the contract you don’t want to pay any more, it is extremely unlikely a company (especially startups) will sue you to enforce the rest of the payments. 
+
+If your contract is usage-based, ensure that any overages have a discount applied as well, rather than the ‘list’ price. This protects you if you get your usage calculations wrong in the first place (which is often).
+
+In terms of the discount itself, you should ask people you know what discount they got (easier), or provide a quote from a competitor (more work). 
```

Comment:
Should we say we've seen 90%+ discounts from list price? I've seen people mention this in the Slack.